### DISCUSSION TITLE
Peridigm for creep

### DISCUSSION THREAD

**Discussion by HappyLingJuan0323** [peridigm/peridigm#318] [Q&A]
How do i use peridigm to simulate creep？

**Comment by oldninja** [comment]
I'm not sure we have a creep model, but if we do, you would like to start at the material interface to understand the parameters. Then go to the unit tests folder and find a simplified test model if it exists.

**Comment by deepakbiitk** [comment]
It depends what type of creep model you are looking for. There is viscoelastic and viscoplastic model available in peridigm.
